Dockerfile

```diff
@@ -168,6 +168,10 @@ RUN pip install --upgrade pip==9.0.1
 RUN pip install -r /opt/requirements.txt
 RUN pip install xvfbwrapper
 
+# install PyPEER
+RUN git clone https://github.com/ChildMindInstitute/PyPEER.git /tmp/PyPEER && \
+    pip install -e /tmp/PyPEER
+
```

Comment:
Shorter:
```
RUN pip install git+https://github.com/ChildMindInstitute/PyPEER.git
```
Installing from /tmp using `-e` is worrying, since /tmp is usually volume binded to docker host and will overwrite the `/tmp`, where pypeer is installed.